frontend/model/state.js

```diff
@@ -388,6 +389,7 @@ const actions = {
     if (settings) {
       console.debug('loadSettings:', settings)
       store.replaceState(settings)
+      captureLogsStart()
```

Comment:
Make sure to add a `captureLogsStart()` to the `else` clause of the `if (settings)`, because otherwise it will not start capturing on first login.